projects/Mallard/src/components/article/html/components/images.ts

```diff
@@ -25,8 +25,8 @@ const breakoutCaption = (
 	.image[data-role='${role}'] div {
 		position: absolute;
 		right: ${px(
-	(metrics.article.rightRail + metrics.article.sides * 1.5) * -1,
-)};
+			(metrics.article.rightRail + metrics.article.sides * 1.5) * -1,
```

Comment:
Don't know what is going on with Prettier here but it refuses to correct it.

Reply:
There are a few of these.

Reply:
Same on my branch, looks fine with yarn prettier but format on save throws it out of line...